src/lesson6_task1/PassengerTransport.java

```diff
@@ -0,0 +1,39 @@
+package lesson6_task1;
+
+public class PassengerTransport extends AirTransport {
+    int countPassengers;
+    boolean isBusinessClass;
+
+    //конструктор
+    public PassengerTransport(int powerHours, int maxSpeed, String brand, float weight,
+                              float wingspan, int minRunwayLength, int countPassengers, boolean isBusinessClass) {
+        super(powerHours, maxSpeed, brand, weight, wingspan, minRunwayLength);
+        this.countPassengers = countPassengers;
+        this.isBusinessClass = isBusinessClass;
+    }
+
+    //возвращает строку с описанием всех характеристик объекта
+    @Override
+    public String getDescription() {
+        this.technicalChar = "Марка " + this.brand + ":\n-Мощность " + this.powerHours + " лошадиных сил;\n-Мощность " + getPowerKw() + " кВ;\n" +
+                "-Максимальная скорость " + this.maxSpeed + " км/ч;\n-Масса " + this.weight + " кг;\n" +
+                "-Размах крыльев " + this.wingspan + " м;\n-Минимальная длина взлётно-посадочной полосы для взлёта " + this.minRunwayLength + " м;\n" +
+                "-Кол-во пассажиров " + this.countPassengers + ";\n-" + checkBusinessClass() + ".\n";
+        return this.technicalChar;
```

Comment:
@Nadya101 сокращаем код :
```suggestion
        return "Марка " + this.brand + ":\n-Мощность " + this.powerHours + " лошадиных сил;\n-Мощность " + getPowerKw() + " кВ;\n" +
                "-Максимальная скорость " + this.maxSpeed + " км/ч;\n-Масса " + this.weight + " кг;\n" +
                "-Размах крыльев " + this.wingspan + " м;\n-Минимальная длина взлётно-посадочной полосы для взлёта " + this.minRunwayLength + " м;\n" +
                "-Кол-во пассажиров " + this.countPassengers + ";\n-" + checkBusinessClass() + ".\n";
```